force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +354,20 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
+        Integer indexOfWord = 0;
 
         // Loop through the list of words
+        for(String currentWord : words){
             // If the current word is the word to find, exit the loop
+            if(currentWord.equals(wordToFind)){
+                return indexOfWord;
```

Comment:
This could pose a problem if the word you're looking for doesn't actually exist in the string of words. Its better to exit the loop early when (if) you find the match and then have the return outside the loop.